src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,29 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```

Comment:
let's move formatter to the class field, since it won't change and we don't need to create new formatters per each method call